backend/spec/models/memo_spec.rb

```diff
@@ -81,4 +81,55 @@
       end
     end
   end
+
+  describe '検索機能のテスト' do
```

Comment:
Idiomatic

テストの対象を具体的にしましょうか！

```suggestion
  describe 'SearchResolver::resolve(memos:, params:)' do
```

Reply:
@ochi-sho-private-study 
修正いたしました。